scitran/data/meeg/meeg.py

```diff
@@ -0,0 +1,190 @@
+"""
+scitran.data.meeg
+=================
+
+Data format for M/EEG data using mne-python.
+
+"""
+
+import logging
+import tempfile
+import zipfile
+import json
+import warnings
+import os
+from os import path as op
+import shutil
+
+from mne.io import read_raw_fif
+
+from .. import data, util
+
+log = logging.getLogger(__name__)  # root logger already configured
+
+# see data.py for expected project properties
+project_properties = data.project_properties
+
+_session_properties = {  # add additional session properties
+    'subject': {
+        'type': 'object',
+        'properties': {
+            'firstname': {
+                'field': 'subj_firstname',
+                'title': 'First Name',
+                'type': 'string',
+            },
+            'lastname': {
+                'field': 'subj_lastname',
+                'title': 'Last Name',
+                'type': 'string',
+            },
+            'dob': {
+                'field': 'subj_dob',
+                'title': 'Date of Birth',
+                'type': 'string',
+                'format': 'date',
```

Comment:
ahhh, based on looking at other modules, I think I need to set the e.g. `.subj_firstname` attribute of the `Reader` class.